app/views/errors/no_draft_page_open.erb

```diff
@@ -0,0 +1,9 @@
+<%= h1 "Error: page not available", size: "xl" %>
+
+<p class="nhsuk-body">
+  You do not have a draft page open, or it has timed out.
```

Comment:
I'm not sure about this wording, it would be the first time we make any reference to users to any concept of "drafts", from the perspective of users I'm not sure they'd understand what that means.